docs/os/security.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Security considerations
+parent: Operating System
+nav_order: 330
+---
+## Security considerations
+
+As a networked device, reachable from the outside, the attack surface of the BitBox Base needs to be minimized as much as possible. This is why we use our BitBox App as the dedicated user interface for node management. By default, only one port is exposed on the Base to communicate over an end-to-end encrypted channel with the App, providing all functionality.
+
+### Pairing Base with BitBox App
+The first step is to safely pair the BitBox Base and App. To achieve this, the Base announces itself on the local network using mDNS and the Middlware exposes provides an API endpoint to initiate a secure connection. The Base is automatically detected by the App and announced within the user interface. 
+
+At first, only a single public API endpoint for pairing is available that allows the two components to establish an encrypted communication channel. To rule out a man-in-the-middle attack, a confirmation code is shown both within the App and on the Base and needs to be confirmed manually. With the secure connection established, the [Noise encryption](http://noiseprotocol.org) can be set up and the full API functionality is available to the App.
```

Comment:
Perhaps "the full API functionality is available to the App" should also mention something about what kind of features are available in "the full API"?

Reply:
I don't think that's necessary in the context of describing our security considerations. What specifically do you have in mind?

Reply:
> What specifically do you have in mind?

If I was not part of the team contributing to this project but I was reading these docs, I'd be curious at this point what "the full API functionality" refers to, so even if described at a high level or through examples, I was suggesting enumerating or otherwise describing some of the features.

Perhaps:

> With the secure connection established, the [Noise encryption](http://noiseprotocol.org) can be set up and the Base API can be accessed by the App, allowing it to view and spend Bitcoin, manage LN channels, etc.
```

Reply:
I added some examples.